react/components/QuoteDetails/QuoteDetails.tsx

```diff
@@ -72,6 +73,7 @@ const QuoteDetails: FunctionComponent = () => {
   const {
     route: { params },
     navigate,
+    workspace,
```

Comment:
You can also grab `accountName` from here instead of from the session. Maybe the user email too, but that one I'm not sure about.